lenovo/legion/16ach6h/nvidia/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, ... }:
```

Comment:
So this module is for nvidia only while the other one is hybrid? Can you have a comment here if this is the case or clarify how this is different from the main module?

Reply:
This module is prepared for the case where ”DDG“ is enabled.
If "DDG" is enabled, then we don't need amdgpu driver, and another reason is that if hardware.nvidia.prime.offload is turned on when "DDG" is turned on, it will cause xorg to crash, but wayland can be run normally